docs/faqs/contributor-faq.md

```diff
@@ -0,0 +1,218 @@
+Contributor FAQ
+===============
+
+The following contains answers to typical questions from contributors about OpenMS.
+
+## General
+
+The following section provides general information to new contributors.
+
+### I am new to OpenMS. What should I do first?
+
+* Check out the development version of OpenMS (see website).
+* Try and build OpenMS according to the installation instructions.
+* Read the [OpenMS Coding Conventions](https://github.com/OpenMS/OpenMS/wiki/Coding-conventions).
+* Read the OpenMS Tutorial.
+* Create a GitHub account.
+* Subscribe to the [open-ms-general](https://sourceforge.net/projects/open-ms/lists/open-ms-general) and [open-ms-developers]() mailing lists.
+  > ***_NOTE:_*** You can see the developers list only if you are logged in to Sourceforge and if you are a OpenMS developer.
+
+### What is the difference between an OpenMS tool and util?
+
+A tool starts its lifecycle in `UTILS` and may exist without being thoroughly tested. Tools may be promoted from `UTILS` to `TOOLS` if they are stable enough, are fully tested, fully documented, and a test workflow exists.
+
+### I have written a class for OpenMS. What should I do?
+
+Follow the [OpenMS coding conventions](coding-conventions.md).
+
+Coding style (brackets, variable names, etc.) must conform to the conventions.
+
+* The class and all the members must be documented thoroughly.
+* You can check your code with the tool  `tools/checker.php`. Call `php tools/checker.php` for detailed instructions.
+
+Please open a pull request and follow the [checklist](pull-request-checklist.md).
```

Comment:
let's prefer saying, `guidelines` everywhere instead of `checklist`, only let the file name include `checklist`.